src/components/FolderName.js

```diff
@@ -0,0 +1,26 @@
+import "./FolderName.css";
+export default function FolderName({ selectedFolder }) {
+  return (
+    <div className="folder-name-box">
+      <div className="folder-name">
+        {selectedFolder ? selectedFolder.name : "전체"}
```

Comment:
```suggestion
        {selectedFolder?.name ?? "전체"}
```
[중요도: 중간] 이런 식으로 optional chaining과 ?? 연산자를 섞어서 쓰면 더 간단하게 사용할 수 있어보이네요~

Reply:
오 더 간단하게 될 수 있군요!